docs/2.0/upgrade.md

```diff
@@ -4,6 +4,17 @@ We generally push changes behind the scenes, so you don't have to update your co
 
 Follow these guides to make sure your configuration files are up to date.
 
+## Upgrade from 2.30.1 to 2.30.2
+We introduced a [security update](https://github.com/avo-hq/avo/pull/1694) that can be affecting several of your `visible` blocks. That blocks are now evaluated before assigning attributes to the models in order to determine if a specific field is or not updatable. Since that evaluation is before assigning the attributes the model is nil when submitting a creation form.
+
+```ruby
+# resource.model is nil when submitting form on resource creation
+visible -> (resource: ) { resource.model }
+```
+
+You can get more context/information [here.](https://github.com/avo-hq/avo/issues/1711)
+
+
```

Comment:
```suggestion
## Upgrade from 2.30.1 to 2.30.2
Following this [security update](https://github.com/avo-hq/avo/pull/1694) the `visible` blocks changed their behavior. The blocks are now evaluated before assigning attributes to the models in order to determine if a specific field is or not `updatable`. Since that evaluation is before assigning the attributes the model is `nil` when submitting a creation form.

```ruby
# `resource.model` is nil when submitting the form on resource creation
field :name, as: :text, visible -> (resource: ) { resource.model.enabled? }

# Do this instead
field :name, as: :text, visible -> (resource: ) { resource.model&.enabled? }
```